go.mod

```diff
@@ -1,6 +1,6 @@
 module github.com/CircleCI-Public/circleci-yaml-language-server
 
-go 1.21
+go 1.22
```

Comment:
doesn't work I replaced this with `go 1.22.0 `to make it work

Reply:
What do you mean by "doesn't work"? This should only affect build, and build seems to work [in the CI](https://app.circleci.com/pipelines/github/CircleCI-Public/circleci-yaml-language-server/1290/workflows/e9044bf6-3f8a-4803-8a3c-ae8d4cc13517/jobs/7873).

Can you describe what doesn't work? Did you install go 1.22 on your machine?

Reply:
@loderunner I think this change is manadatory for LS to build